android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
```

Comment:
Thanks for doing this @nithindaskc67  It would probably make sense to update the example as well: https://github.com/Muljin/msal-flutter/blob/master/example/android/gradle/wrapper/gradle-wrapper.properties#L6 what do you think?  
@mswehli Do you have any plans to review this in the near future?